frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableRow/ticketsTableRow.component.tsx

```diff
@@ -38,22 +49,61 @@ export const TicketsTableRow = ({ ticket, onClick }: { ticket: ITicket, onClick:
 		dueDate,
 	} = getPropertiesInCamelCase(properties);
 
+	const teamspace = TeamspacesHooksSelectors.selectCurrentTeamspace();
+	const ownerAsUser = UsersHooksSelectors.selectUser(teamspace, owner);
+
 	const {
 		treatmentStatus,
 		levelOfRisk,
 	} = getPropertiesInCamelCase(modules?.safetibase || {});
 
+	const handleClick = (e) => {
+		e.preventDefault();
+		onClick();
+	};
+
 	return (
-		<Row key={id} onClick={onClick}>
-			<Highlight search={query}>{`${template.code}:${number}`}</Highlight>
-			<Highlight search={query}>{title}</Highlight>
-			<span>{assignees?.length || 0}</span>
-			<span>{owner}</span>
-			<span>{dueDate ? new Date(dueDate).toLocaleDateString() : 'unset'}</span>
-			<span>{priority}</span>
-			<span>{status}</span>
-			<span>{levelOfRisk}</span>
-			<span>{treatmentStatus}</span>
+		<Row key={id} onClickCapture={handleClick}>
+			<Cell>
+				<Highlight search={query}>{`${template.code}:${number}`}</Highlight>
+			</Cell>
+			<Cell>
+				<Tooltip title={title}>
+					<OverflowContainer>
+						<Highlight search={query}>
+							{title}
+						</Highlight>
+					</OverflowContainer>
+				</Tooltip>
+			</Cell>
+			<Cell>
+				{assignees?.length && (<AssigneesSelect value={assignees} multiple disabled />)}
+				{!assignees?.length && (
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/62481250/419cb38e-ca98-42d8-bd5d-414c6df75bb3)
If the assignees is an empty array the component is prefaced by a "0". 

Also the ticket above doesn't have an assignees key in the ticket object (hence it doesn't have this problem). It is the same template, so this points to another problem in that I think the property isn't added to the redux state until it is edited.